activerecord/test/models/cat.rb

```diff
@@ -1,10 +1,8 @@
 class Cat < ActiveRecord::Base
   self.abstract_class = true
 
-  enum gender: [:female, :male]
-
-  scope :female, -> { where(gender: genders[:female]) }
-  scope :male, -> { where(gender: genders[:male]) }
+  scope :female, -> { where(gender: :female) }
+  scope :male, -> { where(gender: :male) }
```

Comment:
Can we simply remove these scopes? The `enum` declaration is creating them already.